lib/fluent/plugin/in_windows_eventlog2.rb

```diff
@@ -417,6 +417,7 @@ def parse_desc(record)
                          r.split(NONE_FIELD_DELIMITER)
                        end
           key = "" if key.nil?
+          key.strip! # parsed key sometimes contain unexpected " ". So remove it.
           key.chop!  # remove ':' from key
```

Comment:
Although it isn't the code in this change, `chop!` without checking seems a bit rough.
How about checking & stripping `:` along with trailing spaces in one liner?

Reply:
Make sense. thanks.